.env.example

```diff
@@ -0,0 +1,5 @@
+MONGO_AUTH_URL=mongodb://mongo/dbName?retryWrites=true&w=majority
+MORALIS_SERVER_URL=SERVER-URL-HERE
```

Comment:
1, Remove redundant/unused variables.
2, Add COVALENT_API_KEY instead

Reply:
COVALENT_API_KEY is set in .env file